examples/minimal/packages/contracts/package.json

```diff
@@ -4,20 +4,19 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "forge clean && forge build",
-    "deploy": "pnpm run initialize && mud deploy",
-    "deploy:testnet": "pnpm run initialize && mud deploy --profile=lattice-testnet",
+    "build": "pnpm run build:mud && pnpm run build:abi && pnpm run build:types",
+    "build:abi": "forge clean && forge build",
+    "build:mud": "mud tablegen && mud worldgen && mud tsgen --configPath mud.config.ts --out ../client-vanilla/src/mud && mud tsgen --configPath mud.config.ts --out ../client-react/src/mud && mud tsgen --configPath mud.config.ts --out ../client-phaser/src/mud",
+    "build:types": "rimraf types && typechain --target=ethers-v5 out/IWorld.sol/IWorld.json",
```

Comment:
we call this step `build:typechain` in other places, can we name that similarly here?